core/src/com/unciv/ui/newgamescreen/NewGameScreen.kt

```diff
@@ -106,8 +107,9 @@ class NewGameScreen: PickerScreen(){
     var newGame:GameInfo?=null
 
     override fun render(delta: Float) {
-        if(newGame!=null){
+        if (newGame != null){
             game.loadGame(newGame!!)
+            UncivGame.Current.setWorldScreen()
```

Comment:
???? why? 
Doesn't loadGame already contain a setWorldScreen call?

Reply:
Indeed. I believe many issue might be resolved by requesting a rendering step in `setWorldScreen` like in the last commit. Let me know if this solution seems better.